Live-Snap/Live-Snap/Controllers/CropWallpaperViewController.swift

```diff
@@ -51,11 +52,14 @@ class CropWallpaperViewController: UIViewController, CropViewControllerDelegate
         view.addSubview(toolbarInstructionsLabel)
         cropViewController.didMove(toParentViewController: self)
     }
-    
+
+
     func cropViewController(_ cropViewController: CropViewController, didFinishCancelled cancelled: Bool) {
         let selectPhotoViewController = SelectPhotoViewController()
         
         System.shared.appDelegate().pageViewController?.setViewControllers([selectPhotoViewController], direction: .reverse, animated: true, completion: nil)
+
+        cropViewController.delegate = nil
```

Comment:
Fixes memory leak

Reply:
you dont need to save cropViewController as class variable. I think that by itself was the reason for the leak, but could be wrong

Reply:
The delegate was causing the memory leak but didn't know i could access cropViewController without saving it as a class variable